Базовая стилизация всех страниц

feofilaktov2105 opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2288093/tasks/8)

:boom: https://htmlacademy-htmlcss.github.io/2288093-sedona-36/4/

feofilaktov2105 committed:
Начинает добавлять базовую стилизацию

feofilaktov2105 committed:
Продолжает добавлять базовую стилизацию

feofilaktov2105 committed:
Добавляет базовую стилизацию на главную страницу

feofilaktov2105 committed:
Продолжает базовую стилизацию

feofilaktov2105 committed:
Добавляет класс button в разметку, селектор button в стили

feofilaktov2105 committed:
Правит стили кнопки и инпутов

feofilaktov2105 committed:
Добавляет базовую стилизацию на страницу каталога в блок фильтров.

feofilaktov2105 committed:
Стилизует кнопки в карточках каталога

- Подробнее
- В избранное

feofilaktov2105 committed:
Добавляет стилизацию кнопок с числом и звезд рейтинга, пагинации

feofilaktov2105 committed:
Добавляет стилизацию секции подписки на рассылку на странице каталога

feofilaktov2105 committed:
Стилизует хлебные крошки на странице каталога

feofilaktov2105 committed:
Стилизует select с Сортировкой гостиниц

feofilaktov2105 committed:
Стилизует кнопки выбора вида отображения карточек

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -0,0 +1,510 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
```
похоже ты не используешь плагин для редактора editorconfig, у тебя в проекте конфиг для него с отступами в два пробела

feofilaktov2105 replied:
Действительно не был установлен. Поставил.

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -0,0 +1,510 @@
+    font-family: "PT Sans";
+    font-size: 18px;
+    line-height: 21px;
+    color: #242424;
```
цвет не тот, там #333333

feofilaktov2105 replied:
Исправил цвет для body на #333333

Kartaviy-arteom commented on index.html:
```diff
@@ -31,56 +34,68 @@
-        <li class="advantages-item">
+        <li class="advantages-item advantages-item-square">
           <h3 class="advantages-title">Небольшая площадь</h3>
           <p class="advantages-description">Все достопримечательности находятся очень близко</p>
```
у этого элемента другой цвет в макете и у аналогичных

feofilaktov2105 replied:
Исправил для элементов белый цвет на #FFFFFF. 
Изначально цвет был введен прописными буквами #ffffff. Из за этого, видимо, не отрабатывался и унаследовался от body #F2F2F2.

feofilaktov2105 replied:
Исправил цвет элементов на корректный в соответствии с макетом (на color: #333333)
![image](https://user-images.githubusercontent.com/121626291/218808638-88ad14ef-c705-42fb-9033-f980dcf2bc68.png)

feofilaktov2105 replied:
Полностью прошёл по обеим страницам через devtools. Где не доглядел - также поправил цвета, font-size, line-height согласно макету.

Kartaviy-arteom reviewed (commented):
Ты немного запутался в оттенках черного их нужно сверить с макетом

feofilaktov2105 committed:
Правит у элементов цвет, font-size, line-height согласно макету. Форматирование.

Правит у элементов цвет, font-size, line-height согласно макету. Форматирование в файле стилей (удаляет лишние пробелы)
